src/lightning_app/cli/lightning_cli_list.py

```diff
@@ -14,3 +15,18 @@ def list_clusters(**kwargs):
     """List your Lightning AI BYOC compute clusters."""
     cluster_manager = AWSClusterManager()
     cluster_manager.list()
+
+
+@click.option(
+    "--cluster-id",
+    "cluster_id",
+    type=str,
+    required=False,
+    default=None,
+    help="Filter apps by associated Lightning AI compute cluster",
+)
+@get_list.command("apps")
+def list_apps(cluster_id: str, **kwargs):
```

Comment:
Is the naming already decided here?

In our vision to [design a CLI](https://www.notion.so/lightningai/CLI-Redesign-92270473ae494a36b0f59db3e325b1d4) for both Lightning and Grid (eventually becoming one and the same), we imagined having a single top level command to retrieve information about resources, called `show`.

I'm ok having a list command, but want to make sure we know how it is going to fit into the larger picture and what else it is going to do in the future.

cc @adam-lightning

Reply:
this is per the BYOC CLI RFC that was reviewed and approved from the right places (see grid-rfcs # 35). The design you linked is for grid, and as far as I know for lightning CLI folks wanted a different approach.

Reply:
reading the RFC this only pertains to showing individual apps, which is different from listing all the apps that are currently running. We will have a different command to fetch details about an individual app.

Reply:
I recall some time ago parts of the discussion about `list` and `show` commands. But don't really know if and where it's documented... 😓 
My take is I'm ok with separate `show` and `list` as long as their intended use is clear i.e. `list` just lists all resources of some type and gives enough information to later "inspect" or `show` more info by using dedicated command.

Reply:
@awaelchli You should have the last call on the CLI API design there. How do you want to proceed ?